client/my-sites/patterns/components/pattern-library/style.scss

```
@@ -4,6 +4,9 @@
 
 .pattern-library__pill-navigation {
 	background: var(--studio-white);
+	position: sticky;
+	top: 0;
+	z-index: 1000;
```

```suggestion
	z-index: 3;
```
AFAICT, there's no need for such a high `z-index`. `3` seems to achieve the same effect